prelims/Program_Demo.R

```diff
@@ -28,13 +28,9 @@ edit_student_attendance <- function(id, new_status)
 # The function accepts an id number and student name and creates a vector
 # using the provided student details. This vector is then added to the
 # student_list list.
-add_student <- function(id, student_name) 
-{
-  student <- c(id, student_name, "Present");
-  student_list[[length(student_list) + 1]] <- student;
-  cat("Student added successfully.")
-  return(student_list)
-}
+add_student <- function(id, student_name) {
+  student <- c(id, student_name, "Present")
+  student_list[[length(student_list) + 1]] <- student}
```

Comment:
Please refer to Section II of the company's coding standards under SYNTAX - Curly Braces (Allman) Documentation, and ensure compliance with it. Make sure to use the updated curly brace format consistently across the entire code to meet the modified standard. Thank you.
![AllmanFormat](https://github.com/letsgoramen/9480-R-final/assets/148677526/d2b4e773-722d-43a3-bf42-2570272191a6)